nva-datamodel-java/src/test/java/no/unit/nva/api/PublicationResponseTest.java

```diff
@@ -13,9 +17,20 @@ class PublicationResponseTest {
     @Test
     void staticConstructorShouldReturnPublicationResponseWithoutUnexpectedLossOfInformation() {
         Publication publication = PublicationGenerator.randomPublication();
-        assertThat(publication,doesNotHaveEmptyValuesIgnoringFields(Set.of("doiRequest", BOOK_REVISION_FIELD)));
+        assertThat(publication, doesNotHaveEmptyValuesIgnoringFields(Set.of(BOOK_REVISION_FIELD,
+                                                                            ALLOWED_OPERATIONS_FIELD)));
         PublicationResponse publicationResponse = PublicationResponse.fromPublication(publication);
-        assertThat(publicationResponse,doesNotHaveEmptyValuesIgnoringFields(Set.of("doiRequest", BOOK_REVISION_FIELD)));
```

Comment:
@brinxmat Could not figure out why "doiRequest" was used here. I tried a RepeatedTest(1000) just in case and it looks like its not needed anymore

Reply:
Prior model versions stacked all info in the publication response